### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Anushka-shukla**
Issue: https://github.com/common-workflow-language/cwlviewer/issues/307
Adding installation steps in ReadMe to get the application running in the local system

## Description
In the ReadMe.MD it's nowhere mentioned that first, you have to clone, and then inside the project directory we have to run: "docker-compose up". This can be a little confusing for the first-timers.

## Motivation and Context
I was trying to get the application running on my system. I downloaded docker and docker-compose as it was upfront mentioned, then it was mentioned to run: docker-compose up. This started giving me the below error:

Can't find a suitable configuration file in this directory or any
parent. Are you in the right directory?
Supported filenames: docker-compose.yml, docker-compose.yaml, compose.yml, compose.yaml"

## Screenshots (if appropriate):
Screenshot from 2021-04-01 10-21-27

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x ] My change requires a change to the documentation.
- [ x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,16 +34,26 @@ If you are a developer, or you want to use the CWL Viewer in a closed environmen
 
 ## Recommended - Running with Docker
 
-This application can be started with [Docker](https://www.docker.com/) and [Docker Compose](https://docs.docker.com/compose/install/).
+This application can be started with [Docker](https://www.docker.com/) and [Docker Compose](https://docs.docker.com/compose/install/) so get them installed already.
```
Comment (mr-c) [inline README.md:37]:
```suggestion
This application can be started with [Docker](https://www.docker.com/) and [Docker Compose](https://docs.docker.com/compose/install/) and Docker Compose is the recommended method of running or developing this codebase.
```

File: README.md [hunk 2]
```diff
@@ -34,16 +34,26 @@ If you are a developer, or you want to use the CWL Viewer in a closed environmen
...
 
-To start CWLViewer exposed on port `8080`, run:
+Then run the following commands to clone the project in your local system and direct to cwlviewer directory.
```
Comment (mr-c) [inline README.md:39]:
```suggestion
Then run the following commands to clone the project in your local system.
```

File: README.md [hunk 3]
```diff
@@ -34,16 +34,26 @@ If you are a developer, or you want to use the CWL Viewer in a closed environmen
...
+   ``` 
+    git clone https://github.com/common-workflow-language/cwlviewer.git
+    cd cwlviewer 
+  ```
+
+In the project directory, to start CWLViewer exposed on port `8080`, run:
 
+    docker-compose up
+     
+The web server will connect to a local host, you'll see the message saying "Tomacat started on port(s):8080". Then you can see the appliaction already running.
```
Comment (mr-c) [inline README.md:50]:
```suggestion
The web server will connect to a local host, you'll see the message saying "Tomacat started on port(s):8080".

To see the locally running CWL Viewer app, isit https://localhost:8080 in your web browser.
```

**Review by mr-c — CHANGES_REQUESTED** [review]
Dear @Anushka-shukla 

Thank you for helping us improve the documentation! I've made a few suggestions.

File: README.md (hunk 3 above)
Comment (Anushka-shukla) [reply README.md]: Done!

File: README.md (hunk 2 above)
Comment (Anushka-shukla) [reply README.md]: Done!

File: README.md (hunk 1 above)
Comment (Anushka-shukla) [reply README.md]: Done!

**Review by mr-c — APPROVED** [review]

**Pull request merged**